packages/office-ui-fabric-react/src/components/SpinButton/SpinButton.tsx

```diff
@@ -4,7 +4,6 @@ import { Label } from '../../Label';
 import { Icon } from '../../Icon';
 import {
   initializeComponentRef,
-  FocusRects,
```

Comment:
Why are we removing `FocusRects` here?

Reply:
Because it's needed. the focus style is done on focus. doesn't have to be keyboard focus. 
in general , i don't want to add `FocusRects` if not needed. that's why you don't see it everywhere where i removed `BaseComponent`